src/screens/Dressroom/Item/ItemDetailScreen.tsx

```
@@ -0,0 +1,158 @@
+      <AppHeader
+        title="아이템 정보"
+        left={<HeaderLeft type="icon" onPress={() => navigation.goBack()} />}
+        right={
+          <HeaderRight
+            type="icons"
+            icons={[
+              <Pressable key="delete" onPress={() => setAlertVisible(true)}>
+                <DeleteIcon />
+              </Pressable>,
+              <Pressable>
+                <EditIcon key="edit" width={18} height={18} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)